### PR TITLE
Changed device name ARM -> CPU

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

**Comment by l-bat** [comment]
@ilya-lavrenov should we change lib name to `libcpuPlugin.so`?

File: modules/arm_plugin/src/arm_executable_network.cpp [hunk 1]
```diff
@@ -36,7 +36,7 @@ ArmPlugin::ExecutableNetwork::ExecutableNetwork(const std::shared_ptr<const ngra
 
 void ArmPlugin::ExecutableNetwork::InitExecutor() {
     if (_cfg._exclusiveAsyncRequests) {
-        _taskExecutor = ExecutorManager::getInstance()->getExecutor("ARM");
+        _taskExecutor = ExecutorManager::getInstance()->getExecutor("CPU");
     } else {
         auto streamsExecutorConfig = InferenceEngine::IStreamsExecutor::Config::MakeDefaultMultiThreaded(_cfg._streamsExecutorConfig);
         streamsExecutorConfig._name = "ArmStreamsExecutor";
```
Comment (ilya-lavrenov) [inline modules/arm_plugin/src/arm_executable_network.cpp:42]: CPUStreamsExecutor

**Comment by ilya-lavrenov** [comment]
> @ilya-lavrenov should we change lib name to `libcpuPlugin.so`?

I think this is not needed.

**Comment by apankratovantonp** [comment]
Remove from /home/apankrat/projects/openvino/inference-engine/plugins/arm/include/arm_plugin/arm_config.hpp:
```c++
DECLARE_ARM_CONFIG_KEY(THROUGHPUT_STREAMS);
```

**Comment by l-bat** [comment]
`smoke_BehaviorTests/CoreThreadingTests.smoke_GetMetric` failed on master branch

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**